packages/cosmic-swingset/lib/ag-solo/vats/bootstrap.js

```diff
@@ -242,6 +249,7 @@ export function buildRootObject(vatPowers, vatParameters) {
 
     return allComparable(
       harden({
+        ...(plugin ? { plugin } : {}),
```

Comment:
I'm anxious about exposing this to the `.local` object, but I think that's the only way to make your dynamic scheme work.

Reply:
Right.  Now, only agents with an access token can get to `.local`.